<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevelSelect</class>
 <widget class="QWidget" name="LevelSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>100</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>64</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/selectLevelLabel.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="menuButton">
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>45</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>135</width>
          <height>45</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:enabled {
	border-image: url(:/images/menu.png);
	background: black;
	border: 6px solid black;
	border-radius: 6px;
}
QPushButton:hover {
	background: purple;
	border: 6px solid purple;
	border-radius: 6px;
}
QPushButton:pressed {
	background: green;
	border: 6px solid green;
	border-radius: 6px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="levelLabel">
        <property name="font">
         <font>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="playButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>45</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>45</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:disabled {
	border-image: url(:/images/start.png);
	background: gray;
	border: 6px solid gray;
	border-radius: 6px;
}
QPushButton:enabled {
	border-image: url(:/images/startActive.png);
	background: black;
	border: 6px solid black;
	border-radius: 6px;
}
QPushButton:hover {
	background: purple;
	border: 6px solid purple;
	border-radius: 6px;
}
QPushButton:pressed {
	background: green;
	border: 6px solid green;
	border-radius: 6px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
